.gitignore

```diff
@@ -64,10 +64,6 @@ pip-wheel-metadata/
 # Mac OSX
 .DS_Store
 
-astropy_healpix/_compiler.c
```

Comment:
Why are these un-ignored? Are they no longer generated?

Reply:
That's right.